geode-core/src/main/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImpl.java

```diff
@@ -268,16 +263,11 @@ private ServerLocationResponse queryLocators(ServerLocationRequest request) {
     Iterator controllerItr = locators.get().iterator();
     ServerLocationResponse response;
 
-    final boolean isDebugEnabled = logger.isDebugEnabled();
```

Comment:
Please keep the isDebugEnabled and if-conditionals. We wrap Log4j's Logger in FastLogger to hit a volatile instead of the multitude of things that Log4j checks. Geode code is optimized for INFO level or above and this optimization increases our region operation throughput substantially when the log level is INFO or higher.

Reply:
Okay, changed the isDebugEnabled and if conditionals back